src/topwidget.cpp

```diff
@@ -13,6 +13,7 @@ TopWidget::TopWidget(QWidget *parent) :
     mp_historyBackAction->setIcon(QIcon(":/icons/back.svg"));
     mp_historyBackAction->setText(gt("back"));
     mp_historyBackAction->setToolTip(gt("back"));
+    mp_historyBackAction->setShortcut(QKeySequence(Qt::ALT + Qt::Key_Left));
```

Comment:
Instead of changing this "local" action, we should create, configure and use the global action.

Reply:
OK, but is that a good idea to put an icon on a global action?

Reply:
Yes, this is the icon associated with the action (the same way than text and tooltip)
If the "use case" of the action is made without icon, it will not be used.